### DISCUSSION TITLE
該如何控制好情緒

### DISCUSSION THREAD

**Discussion by bestbanana295** [1101-GS4719-C/Python-Community-GS4719-C#1457] [General]
相信大家生活當中常常會有各式個樣不同的事情發生，而其不全然是歡喜事，也有可能是歹事。想請問大家在發生歹事時是怎麼樣去面對的呢 (e.g. Python code 寫不出來)，我自己的話可能會花時間去查明自己到底做錯了甚麼，但是此時情緒還沒有完全冷靜下來所以效果有限，待冷靜下來之後才會去好好檢討自己。不知道大家怎麼想的呢？

**Comment by Alexgns** [comment]
留意自己的情緒變化，了解何時和為何情緒起伏，有助於更好地掌握情緒的控制，與親友交流、運動、寫日記等活動也有助於情緒的平衡和控制。

**Comment by brian431** [comment]
我是還好，不要把事情都看得太嚴重就不太會有這種問題（除非真的是超嚴重）。很多事情其實回頭看都沒有想像中那麼重要，所以當下就冷靜地好好思考解決方法就好。即使出錯了從長遠來看也不是那麼重要。

**Comment by JessyCheng2003** [comment]
我覺得去洗澡很有幫助，尤其是心情不好、想不出來或是頭腦打結的時候，洗澡完會有煥然一新的感覺～另外去跑步其他運動，都可以幫助歐！

**Comment by alicia0725** [comment]
我自己是一個很容易情緒過激的人，所以我的作法是會找一個跟這件事完全不相關的人抱怨，抱怨完冷靜下來才有辦法好好的檢討自己到底錯在哪裡，也比較不會用情緒化的語句傷害別人

**Comment by pudding0803** [comment]
關於這點，我認為可以先冷靜分析這種情況發生的原因，自己跟別人的差別到底在哪裡，說不定其實其他人都只是沒有讓你知道他們有過這段過程而已。

無論是什麼事情，只要會讓我覺得情緒開始不穩時，我就會開始思考自己當下衝動的反應會不會有什麼問題，如果認為不好的話我就會想辦法冷靜然後避免，盡量當一個和平的人🥹

或是先放下這件事情，去稍微休息、思考一下應該也不錯，加油🥳

**Comment by ALAN30359704** [comment]
我個人是在壓力大時，情緒會很不穩定，所以我認為最好抒壓的方法就是把事情做完，也就是把影響情緒的事物處理掉。

**Comment by alstinlin** [comment]
我也是在事情比較多，或壓力比較大的時候會有情緒起伏，所以通常我會先轉移注意力到其他事情上，比如說吃東西、運動等等，等心情平復之後再回去想想應該怎麼處理會比較好，這樣做之後我覺得先前的問題也比較容易克服~

**Comment by aa103967622** [comment]
我習慣把情緒藏起來自己內化，但是這種方式一旦沒有釋放的地方或方法，就危險得多，因為你不知道這些情緒哪一天會累積到無法控制的地步，那會導致你牙起來！通常散散步會很好舒發這些情緒，而且在晚上效果更好，找條幽靜的小路走一走，當下你就會回去思考最近發生不如意的事情，然後自然你就會放下那些事情，大事化小，如此就內化成功了。

**Comment by xiang305** [comment]
我在自己的情緒不穩定時，我會先反問自己，我在這邊憤怒或失落對事情能有什麼幫助，然後再透過冥想整理自己的情緒，讓自己能夠平靜且理性地來處理事情

**Comment by quan0715** [comment]
我們要學會控制好情緒，了解自己情緒狀態，深呼吸放鬆，找適合的調節方法，尋求他人支持，注重身心健康，接受情緒起伏，持續學習成長。
千萬不要被情緒帶著走，有時候一時的衝動會對不管自己還是周圍的人造成永久性的傷害

**Comment by Aaron0810** [comment]
我在發現自己情緒不好的時候，會喜歡自己一個人獨處，有人來關心我我反而會更不開心，我比較喜歡一個人靜靜，甚至會在學校裡面自己去隨意走走，或是探索一些學校沒去過的地方來轉換心情，沉澱一下在回到該面對的事情，這樣會好蠻多的，不過這當然是我自己的看法而已，也是有人喜歡被安慰的。

**Comment by MaxChi870** [comment]
當遇到煩人或不順心的事時，我都會透過打遊戲或觀看影片來轉移注意力，藉此將自己拉出那個負面的情緒環境，其他的休閒或放鬆活動，甚至是放空，也都是不錯的方式。

**Comment by Alex11392** [comment]
我覺得轉移注意力去其他事情上可以有效的幫助自己冷靜。

**Comment by ThanatosJun** [comment]
當我情緒不好的時候通常有三種解決方式，第一種就是打球放鬆，肌肉的刺激讓我情緒往亢奮移動，而不是往憤怒轉移；二是在深夜中獨自一個人待著，思考一下看看夜空，自然的聲音令人療育；三是跟摯友交流，因為是摯友不必管甚麼外在形象，想講甚麼就講甚麼，情緒彼此抒發。
